chpl/chpl-service/src/main/java/gov/healthit/chpl/logging/LoggingFilter.java

```diff
@@ -0,0 +1,23 @@
+package gov.healthit.chpl.logging;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.apache.logging.log4j.ThreadContext;
+import org.springframework.web.filter.GenericFilterBean;
+
+public class LoggingFilter extends GenericFilterBean {
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+
+        ThreadContext.put("tracker", UUID.randomUUID().toString());
```

Comment:
Is this part of this ticket? I had thought we were going towards using Datadog for this logging stuff. I'm not sure if this stuff should still be here. Not sure about the other `@Loggable` annotations though; if those are Datadog then that's maybe fine, but not sure

Reply:
I had similar questions about the Loggable changes. If we are proceeding with it I would like a technical overview so I know which classes/methods to add it to, what we are expecting to do with it, are there test cases for the Loggable changes, etc.

Reply:
This particular change should be removed.  I think the `@Loggable` should remain, though.  The `@Loggable` annotation on a class will write out for each public method called 1) that the method was called, 2) that the method completed, 3) how long the method took.  I think this can be expanded to also include some error logging. I would actually like to expand this to nearly all of our classes.  As far as a unique token to track which calls where made by which request - this will be injected by some DataDog via some java startup params.  Thoughts?